lib/datasource/index.spec.ts

```diff
@@ -32,9 +32,8 @@ describe(getName(__filename), () => {
   });
   it('returns datasources', () => {
     expect(datasource.getDatasources()).toBeDefined();
-    expect(datasource.getDatasourceList()).toBeDefined();
```

Comment:
Why this change?

Reply:
I think it was a workaround when I had directly imported the datasources in `index.ts`. Reverted.